helpers/z_schema.js

```diff
@@ -159,22 +159,22 @@ var liskFormats = {
 	 * @returns {boolean}
 	 * @todo Add description for the function, the params and the return value
 	 */
-	encryptedSecret(str) {
+	encryptedPassphrase(str) {
 		// Explanation of regex structure:
 		// - 1 or more 'key=value' pairs delimited with '&'
 		// Examples:
-		// - encryptedSecret=abcd1234
-		// - encryptedSecret=abcd1234&iterations=10000&iv=ef012345
+		// - encryptedPassphrase=abcd1234
+		// - encryptedPassphrase=abcd1234&iterations=10000&iv=ef012345
 		// NOTE: Maximum lengths chosen here are arbitrary
 		const keyRegExp = /[a-zA-Z0-9]{2,15}/;
 		const valueRegExp = /[a-f0-9]{1,256}/;
 		const keyValueRegExp = new RegExp(
 			`${keyRegExp.source}=${valueRegExp.source}`
 		);
-		const encryptedSecretRegExp = new RegExp(
+		const encryptedPassphraseRegExp = new RegExp(
 			`^(${keyValueRegExp.source})(?:&(${keyValueRegExp.source})){0,10}$`
 		);
-		return encryptedSecretRegExp.test(str);
+		return encryptedPassphraseRegExp.test(str);
```

Comment:
The final regex here become,

```
^([a-zA-Z0-9]{2,15}=[a-f0-9]{1,256})(?:&([a-zA-Z0-9]{2,15}=[a-f0-9]{1,256})){0,10}$
```

I tried to match 
```
encryptedPassphrase=abcd1234&iterations=10000&iv=ef012345
var1=john&var2=robert&var3=carrell
```

And it didn't worked. Can you double check the reg expression.

Reply:
Great catch. `{2,15}` was set relative to the length of `encryptedSecret`, but `encryptedPassphrase` is longer.

Reply:
`encryptedPassphrase` never appear as key, I updated function description.